.github/workflows/layer_rename.yml

```diff
@@ -0,0 +1,161 @@
+# Rename Layer
+# ---
+# This workflow copies a specific layer version in an AWS account, renaming it in the process
+#
+# Using a matrix, we pull each architecture and python version of the layer and store them as artifacts
+# we upload them to each of the AWS accounts.
+#
+# A number of safety checks are performed to ensure safety.
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: Deployment environment
+        type: choice
+        options:
+          - beta
+          - prod
+        default: Gamma
+        required: true
+      version:
+        description: Layer version to duplicate
+        type: number
+        required: true
+  workflow_call:
+    inputs:
+      environment:
+        description: Deployment environment
+        type: string
+        default: Gamma
+        required: true
```

Comment:
Do we need this?

Reply:
Yes – we still use dynamic environment loading